components/builder-jobsrv/src/data_store.rs

```diff
@@ -416,12 +431,20 @@ impl DataStore {
     ) -> Result<jobsrv::ProjectJobsGetResponse> {
         let conn = self.pool.get_shard(0)?;
         let rows = &conn.query(
-            "SELECT * FROM get_jobs_for_project_v1($1)",
-            &[&(project.get_name())],
+            "SELECT * FROM get_jobs_for_project_v2($1, $2, $3)",
+            &[
+                &(project.get_name()),
+                &project.limit(),
+                &(project.get_start() as i64),
+            ],
         ).map_err(Error::ProjectJobsGet)?;
 
-        let mut response = jobsrv::ProjectJobsGetResponse::new();
         let mut jobs = protobuf::RepeatedField::new();
+        let mut response = jobsrv::ProjectJobsGetResponse::new();
+        response.set_start(project.get_start());
+        response.set_stop(self.last_index(project, &rows));
+        response.set_count(rows.len() as u64);
```

Comment:
I thought this was intended to be the total count of all rows in the db table but it looks like this is just returning the number of rows in this result set.

Reply:
Ah yes, my bad.  I'll fix that up.